source/Octopus.Client/Model/PropertyValueResource.cs

```diff
@@ -75,7 +75,8 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                                 new SensitiveValue
                                 {
                                     HasValue = jo.GetValue("HasValue")?.ToObject<bool>() ?? false,
-                                    NewValue = jo.GetValue("NewValue").ToObject<string>()
+                                    NewValue = jo.GetValue("NewValue").ToObject<string>(),
```

Comment:
I'm surprised this wasn't jo.GetValue("NewValue")**?**.ToObject<string>(), given the following line has that. Or doesn't the following not need it either?

Reply:
We need Elvis for `Hint` because the property won't be there for older Servers. I could make it the same for `NewValue` but I was inclined to leave it alone.